### ISSUE TITLE
Error with Contao 4.8

### ISSUE THREAD

**Issue by akroii** [1up-lab/ContaoSecurityCheckerBundle#10]
I've updated Contao 4.7 to 4.8 and I'am getting this error in Manager Console:

```
Confirm & Close
> Resolving dependencies using Composer Cloud v2.0.11

# Job ID J138kXzneipeYxiTR1TaZ0OHHs5ZxKSSqnIH9KeA3HVAyd8HJSbcItAdlNNTfZiSQ62MusvBtumFGYBYkAO4i7KDKOvC8noUta9pE1 completed in 58 seconds
# Memory usage: 820.4MB (peak: 3524.16MB), time: 56.93s.

$ /opt/php7.3/bin/php '-q' '/var/www/w35gbmh/htdocs/contao/web/contao-manager.phar.php' 'composer' 'install' '--prefer-dist' '--no-dev' '--no-progress' '--no-suggest' '--no-ansi' '--no-interaction' '--optimize-autoloader' 2>&1

Loading composer repositories with package information
Installing dependencies from lock file
Package operations: 6 installs, 37 updates, 6 removals
  - Removing terminal42/header-replay-bundle (1.5.3)
  - Removing sonata-project/google-authenticator (2.2.0)
  - Removing contao-components/respimage (1.4.2)
  - Removing contao-components/mediaelement (4.2.10)
  - Removing contao-components/html5shiv (3.7.3)
  - Removing contao-components/compass (0.12.2.1)
As there is no 'unzip' command installed zip files are being unpacked using the PHP zip extension.
This may cause invalid reports of corrupted archives. Besides, any UNIX permissions (e.g. executable) defined in the archives will be lost.
Installing 'unzip' may remediate them.
  - Installing symfony/polyfill-php73 (v1.12.0): Downloading (100%)
  - Installing symfony/mime (v4.3.3): Downloading (100%)
  - Updating symfony/http-foundation (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/event-dispatcher (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/http-kernel (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/dependency-injection (v4.2.11 => v4.3.3): Downloading (100%)
  - Installing terminal42/service-annotation-bundle (1.0.0): Downloading (100%)
  - Updating symfony/yaml (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/twig-bridge (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/filesystem (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/config (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/twig-bundle (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/translation (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating egulias/email-validator (2.1.10 => 2.1.11): Downloading (100%)
  - Updating symfony/security (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/security-bundle (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/routing (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/process (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/lock (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/finder (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/cache (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/framework-bundle (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/expression-language (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/console (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/asset (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating scheb/two-factor-bundle (v3.22.0 => v4.5.0): Downloading (100%)
  - Installing doctrine/instantiator (1.2.0): Downloading (100%)
  - Installing doctrine/common (v2.10.0): Downloading (100%)
  - Installing doctrine/orm (v2.6.3): Downloading (100%)
  - Updating contao/image (0.3.9 => 1.0.0): Downloading (100%)
  - Updating contao-components/tablesort (3.4.12 => 4.0.1): Downloading (100%)
  - Updating contao-components/contao (8.0.3 => 9.0.0): Downloading (100%)
  - Updating contao/core-bundle (4.7.7 => 4.8.0): Downloading (100%)
  - Updating alnv/catalog-manager (v1.31.7 => v1.31.9): Downloading (100%)
  - Updating contao/calendar-bundle (4.7.7 => 4.8.0): Downloading (100%)
  - Updating symfony/web-profiler-bundle (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating ocramius/proxy-manager (2.2.2 => 2.2.3): Downloading (100%)
  - Updating symfony/proxy-manager-bridge (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/monolog-bridge (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/dotenv (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating symfony/debug-bundle (v4.2.11 => v4.3.3): Downloading (100%)
  - Updating contao/installation-bundle (4.7.7 => 4.8.0): Downloading (100%)
  - Updating contao/manager-bundle (4.7.7 => 4.8.0): Downloading (100%)
Package heimrichhannot/contao-fieldpalette is abandoned, you should avoid using it. Use heimrichhannot/contao-fieldpalette-bundle instead.
Package phpoffice/phpexcel is abandoned, you should avoid using it. Use phpoffice/phpspreadsheet instead.
Generating optimized autoload files
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
contao/manager-plugin: Generating plugin class...
contao/manager-plugin: ...done generating plugin class
> Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication

In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:
                                                                               
  The service "oneup_contao_security_checker.notification_listener" has a dep  
  endency on a non-existent service "templating".                              
                                                                               

Script Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication handling the post-install-cmd event terminated with an exception

In ScriptHandler.php line 89:
                                                                               
  An error occurred while executing the "contao:install-web-dir" command:      
  In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:                 
                                                                               
                                                                               
    The service "oneup_contao_security_checker.notification_listener" has a d  
  ep                                                                           
    endency on a non-existent service "templating".                            
                                                                               

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...


# Process terminated with exit code 1
# Result: General error


---

### The background task has stopped unexpectedly ###
```

My composer.json:
```
{
    "type": "project",
    "require": {
        "alnv/catalog-manager": "^1.30",
        "alnv/catalog-manager-export": "^1.0",
        "alnv/catalog-manager-importer": "^1.2",
        "alnv/catalog-manager-switch": "^1.0",
        "contao/calendar-bundle": "^4.7",
        "contao/conflicts": "*@dev",
        "contao/manager-bundle": "4.8",
        "derhaeuptling/contao-lazy-images": "^3.0",
        "fritzmg/contao-lazy-load-element": "^1.0",
        "hofff/contao-shariff": "^3.0",
        "oneup/contao-security-checker-bundle": "^0.4.5",
        "richardhj/contao-ajax_reload_element": "^2.1",
        "trilobit-gmbh/contao-headerfootercode-bundle": "^1.0"
    },
    "extra": {
        "contao-component-dir": "assets"
    },
    "scripts": {
        "post-install-cmd": [
            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
        ],
        "post-update-cmd": [
            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
        ]
    }
}
```

**Comment by bytehead** [comment]
Can you give a try with version `0.4.8`?

**Comment by akroii** [comment]
Nope:

```
$ /opt/php7.3/bin/php '-q' '/var/www/w35gbmh/htdocs/contao/web/contao-manager.phar.php' 'composer' 'require' 'oneup/contao-security-checker-bundle' '--no-suggest' '--no-update' '--no-scripts' '--prefer-stable' '--sort-packages' '--no-ansi' '--no-interaction' 2>&1

Using version ^0.4.8 for oneup/contao-security-checker-bundle
/var/www/w35gbmh/htdocs/contao/composer.json has been updated

# Process terminated with exit code 0
# Result: OK


> Resolving dependencies using Composer Cloud v2.0.11

# Job ID OMm1bRqJuDHW07Ex8zaES1i4d3JXWBRSpqNQnNZ4I2GSa456jprpEUDynzDGmPXZHNSTcNKA completed in 69 seconds
# Memory usage: 752.56MB (peak: 3391.92MB), time: 68.1s.

$ /opt/php7.3/bin/php '-q' '/var/www/w35gbmh/htdocs/contao/web/contao-manager.phar.php' 'composer' 'install' '--prefer-dist' '--no-dev' '--no-progress' '--no-suggest' '--no-ansi' '--no-interaction' '--optimize-autoloader' 2>&1

Loading composer repositories with package information
Installing dependencies from lock file
Package operations: 2 installs, 0 updates, 0 removals
As there is no 'unzip' command installed zip files are being unpacked using the PHP zip extension.
This may cause invalid reports of corrupted archives. Besides, any UNIX permissions (e.g. executable) defined in the archives will be lost.
Installing 'unzip' may remediate them.
  - Installing sensiolabs/security-checker (v5.0.3): Loading from cache
  - Installing oneup/contao-security-checker-bundle (0.4.8): Loading from cache
Package heimrichhannot/contao-fieldpalette is abandoned, you should avoid using it. Use heimrichhannot/contao-fieldpalette-bundle instead.
Package phpoffice/phpexcel is abandoned, you should avoid using it. Use phpoffice/phpspreadsheet instead.
Generating optimized autoload files
ocramius/package-versions: Generating version class...
ocramius/package-versions: ...done generating version class
contao/manager-plugin: Generating plugin class...
contao/manager-plugin: ...done generating plugin class
> Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication

In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:
                                                                               
  The service "oneup_contao_security_checker.notification_listener" has a dep  
  endency on a non-existent service "templating".                              
                                                                               

Script Contao\ManagerBundle\Composer\ScriptHandler::initializeApplication handling the post-install-cmd event terminated with an exception

In ScriptHandler.php line 89:
                                                                               
  An error occurred while executing the "contao:install-web-dir" command:      
  In CheckExceptionOnInvalidReferenceBehaviorPass.php line 86:                 
                                                                               
                                                                               
    The service "oneup_contao_security_checker.notification_listener" has a d  
  ep                                                                           
    endency on a non-existent service "templating".                            
                                                                               

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...


# Process terminated with exit code 1
# Result: General error


---

### Der Hintergrundprozess wurde unerwartet beendet ###
```

**Comment by bytehead** [comment]
It's funny, because it runs on my contao demo. can you paste your `composer.json`?

**Comment by akroii** [comment]
```
{
    "type": "project",
    "require": {
        "alnv/catalog-manager": "^1.30",
        "alnv/catalog-manager-export": "^1.0",
        "alnv/catalog-manager-importer": "^1.2",
        "alnv/catalog-manager-switch": "^1.0",
        "contao/calendar-bundle": "^4.7",
        "contao/conflicts": "*@dev",
        "contao/manager-bundle": "4.8",
        "derhaeuptling/contao-lazy-images": "^3.0",
        "fritzmg/contao-lazy-load-element": "^1.0",
        "hofff/contao-shariff": "^3.0",
        "oneup/contao-cache-warmup": "^2.1",
        "oneup/contao-security-checker-bundle": "^0.4.8",
        "richardhj/contao-ajax_reload_element": "^2.1",
        "trilobit-gmbh/contao-headerfootercode-bundle": "^1.0",
        "trilobit-gmbh/contao-serverhint-bundle": "^1.0"
    },
    "extra": {
        "contao-component-dir": "assets"
    },
    "scripts": {
        "post-install-cmd": [
            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
        ],
        "post-update-cmd": [
            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
        ]
    }
}
```

**Comment by bytehead** [comment]
Thanks, reproducable. I'll check.

**Comment by bytehead** [comment]
Are you using the Contao Manager?

**Comment by akroii** [comment]
Yes. Latest Version

**Comment by bytehead** [comment]
Fixed in `0.4.9`.
